.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "files.associations": {
+        "*.sql": "oracle",
```

Comment:
SQL / Oracle in InfiniTime ?

Reply:
strange... I installed same extensions for another project, VSC making stange things.